core/src/reporter/state.ts

```diff
@@ -276,10 +283,25 @@ export class State {
     return reporters
   }
 
+  async getRemoteNonce(wallet: Wallet): Promise<number> {
```

Comment:
@Intizar-T I think we could use this function inside of `refresh` as well.

https://github.com/Bisonai/orakl/blob/03d4bd22e901af24dc6bbaa17604f38bebc38d5c/core/src/reporter/state.ts#L258

Reply:
since i've removed fetching remote nonce logic from local nonce manager, this function has been removed as well